leptonai/api/v1/ingress.py

```diff
@@ -1 +1,100 @@
 # todo
+from typing import Union
+from .api_resource import APIResourse
+from .types.deployment import LeptonDeployment
+from .types.deployment_operator_v1alpha1.ingress import LeptonIngressEndpoint
+from .types.ingress import LeptonIngress
+
+
+class IngressAPI(APIResourse):
+    def _to_name(self, name_or_ingress: Union[str, LeptonIngress]) -> str:
```

Comment:
nit: we probably don't need _to_deployment_name below, but just need a to_name that has LeptonDeployment added to the name_or_ingress type hint.